README.md

```diff
@@ -1,16 +1,26 @@
-# ARTwarp 
-
-ARTwarp is a MATLAB-based programme for the automated categorisation of tonal
-animal sounds. It has been tested successfully for bottlenose dolphin whistles
-and killer whale calls, but will be applicable to any sound that can be
-described by frequency contours.
-
-These `ARTwarp.m` files have been run successfully using Matlab 2016a and 2016b
-
-`ARTwarp.m` will open whistle contours extracted using Beluga
-
-`ARTwarp_txt.m` provides the option to open `.ctr` (Beluga), `.csv`, or `.txt` files
-
-This version of `ARTwarp_txt` displays the iteration number and number of whistles 
-analysed in the command window while ARTwarp is running. It saves the results at 
-the end of each iteration.
+# ARTwarp 
+
+ARTwarp is a MATLAB-based programme for the automated categorisation of tonal
+animal sounds. It has been tested successfully for bottlenose dolphin whistles
+and killer whale calls, but will be applicable to any sound that can be
+described by frequency contours.
+
+These `ARTwarp.m` files have been run successfully using Matlab 2016a and 2016b
+
+`ARTwarp.m` will open whistle contours extracted using Beluga
+
+`ARTwarp_txt.m` provides the option to open `.ctr` (Beluga), `.csv`, or `.txt` files
+
+This version of `ARTwarp_txt` displays the iteration number and number of whistles 
+analysed in the command window while ARTwarp is running. It saves the results at 
+the end of each iteration.
+
+
+## Old Classes

```

Comment:
I wonder if this is the right way to do this. Maybe you should have started another branch instead? Michael's code will be there under version control anyway at least for a while, until we decide to keep that  branch or delete it)